api/external/nodes/manager/manager.proto

```diff
@@ -150,6 +220,13 @@ service NodeManagerService {
 			action: "infra:nodeManagers:list"
 		};
 	};
+
+	/*
+	Connect
+
+	Attempts to reach the API for the given nodemanager id to validate the 
+	credentials associated with the nodemanager.
+	*/
 	rpc Connect(Id) returns (ConnectResponse) {
```

Comment:
I'm not sure if there's anything we can do, but some text for the response here would be nice like "If the credentials are bad you'll get an error, an empty response means its fine." It currently looks like this:

![Screen Shot 2020-02-14 at 11 11 11 AM](https://user-images.githubusercontent.com/22962/74560119-b99c6580-4f1a-11ea-87f6-13969bf1a0cc.png)